app/api/helpers/order.py

```diff
@@ -277,6 +286,8 @@ def calculate_order_amount(tickets, discount_code=None):
                 'discount': discount_data,
                 'ticket_fee': round(ticket_fee, 2),
                 'sub_total': round(sub_total, 2),
+                'ticket_tax': round(ticket_tax, 2),
+                'discounted_tax': round(discounted_tax, 2)
```

Comment:
changes are ok. But I wanted to ask, what if we only keep one 'ticket_tax' and only show it there. As when there will be discount code, we will update it, so we have to only keep track of one tax variable. are there any problems using this approach?

Reply:
As the idea is to move all the tax calculations to one place, we may lose track of the original tax value on applying discount code if we use only one variable as we need both in frontend at once.

Reply:
cool. So this PR is completed or anything else need to be done here?

Reply:
Yes. This is complete. But please do not merge it until #7871 's pr gets completed.